haystack/pipeline.py

```diff
@@ -593,6 +598,43 @@ def run(self, **kwargs):
         return kwargs, "output_1"
 
 
+class QueryClassifier(BaseComponent):
+    """
+    A node to choose between two nodes based on query classification result.
+    """
+
+    outgoing_edges = 2
+
+    def __init__(
+        self, 
+        query_classifier: Optional[GradientBoostingClassifier] = pickle.load(
```

Comment:
You can make it None and initialise inside the constructor. As this style is being followed in this project.
```
query_classifier: Optional[GradientBoostingClassifier] = None,
query_vectorizer: Optional[TfidfVectorizer] = None
```